blockchain/system/kip113_test.go

```diff
@@ -74,13 +79,22 @@ func TestAllocKip113(t *testing.T) {
 		nodeId2       = common.HexToAddress("0xbbbb")
 		_, pub1, pop1 = makeBlsKey()
 		_, pub2, pop2 = makeBlsKey()
+
+		abi, _   = contracts.KIP113MockMetaData.GetAbi()
+		input, _ = abi.Pack("initialize")
+
+		allocProxyStorage  = AllocProxy(KIP113MockAddr)
```

Comment:
Do you think `TestAllocKip113` should test proxy alloc as well? This means that for all future system contracts, TestAlloc** will have to test proxy alloc. Does separating ProxyAlloc test in `proxy_test.go` make sense?

Reply:
I didn't try to "test" the proxy but just want to use them like actual deployment. 

The reason that I explicitly separated `AllocProxy` is that I thought it looked more natural rather than putting them all in `AllocKip113`. If you have any idea about this, please tell me.

IMO, I don't think we need a separate `proxy_test`.